_sources/Selection/PrecedenceofOperators.rst

```diff
@@ -55,6 +55,19 @@ Level     Category        Operators
 
    Which of the following properly expresses the precedence of operators (using parentheses) in the following expression: 5*3 > 10 and 4+6==11
 
+Here is an animation for the above expression:
+
+.. showeval:: se_tq631
+   :trace_mode: true
+
+   5 * 3 > 10 and 4 + 6 == 11
+   ~~~~
+   {{5 * 3}}{{15}} > 10 and 4 + 6 == 11
+   15 > 10 and {{4 + 6}}{{10}} == 11
```

Comment:
I think I would add a step here:
`{{15 > 10}}{{True}} and 4 + 6 == 11`
This would make it more explicit that the first clause of the and is evaluated before going on to the second.